example/Components/AdvancedInstantValidation.js

```diff
@@ -8,7 +8,7 @@ export default class AdvancedInstantValidation extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = {form: { name: '', password: '', repeatPassword: '', random: '' }}
+    this.state = {form: { name: '', password: '', repeatPassword: '', random: '', comment: '' }}
```

Comment:
`comment` is added as a validated field that is not considered required (less than 140 char).